README.md

```diff
@@ -1,2 +1,238 @@
-# unmock-python
-Python client for Unmock
+# Unmock (Python SDK)
+[![CircleCI](https://circleci.com/gh/unmock/unmock-python.svg?style=svg)](https://circleci.com/gh/unmock/unmock-python)
+
+Public API mocking for Python.
+
+Unmock can be used to test modules that perform requests to third-party APIs like Hubspot, SendGrid, Behance, and
```

Comment:
Will we have API documentation for `unmock-python` (like readthedocs) and would this belong there (as well)?

Reply:
Good